src/Sets/SparsePolynomialZonotope.jl

```diff
@@ -288,6 +288,37 @@ function rand(::Type{SparsePolynomialZonotope};
     return SparsePolynomialZonotope(center(SSPZ), genmat(SSPZ), GI, expmat(SSPZ))
 end
 
+
+"""
+    exact_sum(P1::SparsePolynomialZonotope, P2::SparsePolynomialZonotope)
+
+Compute the exact sum of sparse polyomial zonotopes ``P₁`` and ``P₂``.
+
+### Input
+
+- `P1` -- sparse polynomial zonotope
+- `P2` -- sparse polynomial zonotope
+
+### Output
+
+exact sum ``P₁ ⊞ P₂``
+
+"""
+function exact_sum(P1::SparsePolynomialZonotope, P2::SparsePolynomialZonotope)
+
+    indexvector(P1) == indexvector(P2) || throw(ArgumentError("Exact sum currently only implemented for Sparse Polynomial Zonotopes with the same index vector"))
+
+    c = center(P1) + center(P2)
+    G = hcat(dependent_genmat(P1), dependent_genmat(P2))
+    GI = hcat(independent_genmat(P1), independent_genmat(P2))
+    E = hcat(expmat(P1), expmat(P2))
+    idx = indexvector(P1)
+
+    return SparsePolynomialZonotope(c, G, GI, E, idx)
```

Comment:
let's leave it like this for now, but it may be better to return after reducing the result (`remove_redundant_generators` or `compact` in Niklas' thesis)

Reply:
fwiw, being defined as alias (with `const`) typing `?\boxplus` currently already returns the docstring of `exact_sum`. After adding an ad-hoc docstring it returns the new one. That is

### Before

```julia
help?> ⊞
"⊞" can be typed by \boxplus<tab>

search: ⊞

  exact_sum(P1::SparsePolynomialZonotope, P2::SparsePolynomialZonotope)


  Compute the exact sum of sparse polyomial zonotopes P₁ and P₂.

  Input
  –––––––

    •  P1 – sparse polynomial zonotope

    •  P2 – sparse polynomial zonotope

  Output
  ––––––––

  exact sum P₁ ⊞ P₂
```

### After
```julia
help?> ⊞
"⊞" can be typed by \boxplus<tab>

search: ⊞

  ⊞(X::SparsePolynomialZonotope, Y::SparsePolynomialZonotope)


  Unicode alias constructor ⊞ (boxplus) for the exact sum operation.

  Notes
  –––––––

  Write \boxplus[TAB] to enter this symbol.
```

this is slightly different by `minkowski_sum`, because `\oplus` technically is not an alias atm and hence it needs its own docstring. Note that when looking for help of an unicode symbol, julia will always by default print how to type that.

Reply:
I added a comment in the issue you opened to remember to check where to call `compact`